src/module/features/chat-alert.ts

```diff
@@ -158,53 +158,6 @@ const ACTOR_TYPE_HANDLERS: ActorTypeHandlers = {
 			}
 		}
 
-		const debilities = [
```

Comment:
Now managed by `IronActiveEffect#_onCreate` and friends. FVTT itself uses `ActiveEffect#_onCreate` for its scrolling token text, which serves a similar purpose.